test/cmdlineTests.sh

```diff
@@ -210,14 +243,14 @@ printTask "Testing unknown options..."
 
 
 printTask "Testing passing files that are not found..."
-test_solc_behaviour "file_not_found.sol" "" "" "" 1 "\"file_not_found.sol\" is not found."
+test_solc_behaviour "file_not_found.sol" "" "" "" 1 "\"file_not_found.sol\" is not found." "" ""
```

Comment:
Why are these filepaths empty?

Reply:
Because they don't have their expected outputs saved as files, we cannot auto-update it.(those 2 empty arguments are `stdout_expectation_file` and `stderr_expectation_file`)

Reply:
Ah, ok. I see.